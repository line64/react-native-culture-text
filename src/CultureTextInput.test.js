import 'react-native';
import React from 'react';
import { View } from 'react-native';
import { CultureContext, TextInput } from '../';

import renderer from 'react-test-renderer';

const translations = {
  es: {
    name_placeholder: 'Escriba su nombre',
  },
}

it('render translated message', () => {
  const tree = renderer.create(
    <CultureContext translations={translations} locale="es">
      <View>
        <TextInput messageKey="name_placeholder" placeholder="Type your name" />
      </View>
    </CultureContext>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
