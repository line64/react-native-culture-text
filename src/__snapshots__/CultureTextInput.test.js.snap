// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render translated message 1`] = `
<View>
  <TextInput
    allowFontScaling={true}
    placeholder="Escriba su nombre"
  />
</View>
`;
